source/kraken/metrics.h

```diff
@@ -31,31 +31,42 @@ www.navitia.io
 #pragma once
 
 #include <memory>
-#include <prometheus/exposer.h>
 #include <map>
-#include "type/type.pb.h"
-#include <boost/optional.hpp>
 
+#include <boost/optional.hpp>
 #include <boost/utility.hpp>
 
+#include "type/type.pb.h"
+
+#if PROMETHEUS_IS_ACTIVED
```

Comment:
This warns when not defined, please choose between:
* always define PROMETHEUS_IS_ACTIVED (to true or false)
* use #ifdef

Reply:
And ENABLE_PROMETHEUS is probably a better name

Reply:
Yes, better
:white_check_mark: